frontend/packages/console-shared/src/components/dropdown/ResourceDropdown.tsx

```diff
@@ -181,6 +187,12 @@ class ResourceDropdown extends React.Component<ResourceDropdownProps, State> {
       sortedList[allSelectorItem.allSelectorKey] = allSelectorItem.allSelectorTitle;
     }
 
+    _.keys(appendItems).forEach((key) => {
+      if (!_.has(unsortedList, key)) {
+        unsortedList = { ...unsortedList, ...{ [key]: appendItems[key] } };
+      }
+    });
```

Comment:
Unnecessary if you init `unsortedList` with `appendItems`

Reply:
Thanks , have updated